### PR TITLE
Network tracers should be owned by the networking team

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: CODEOWNERS
Comment (Jimbo4350) [inline CODEOWNERS:11]: I'm happy to give you merge permissions on `cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs` but if something needs to be merged urgently otherwise you can ping one of the codeowners

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]

File: CODEOWNERS
Comment (coot) [reply CODEOWNERS:11]: My intentions here isn't about merging things quickly; I'd prefer if the node team also reviews our PRs (the more eyes on it the better), but rather making sure that we are notified and asked to provide a review when other teams are making changes.

Comment (coot) [reply CODEOWNERS:11]: @Jimbo4350 ping

Comment (Jimbo4350) [reply CODEOWNERS:11]: Sorry, we've have problems in the past with people merging things in the node without node teams approval and we want to enforce this.

Comment (coot) [reply CODEOWNERS:11]: > You can subscribe to the node to be notified about PRs and you don't need this specific change to be automatically asked to review a PR that makes changes to cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

This does not work, because the updates from `cardano-node` are overwhelming. But we can relay on your judgment to require a review from us if things are related to the networking (tracers, network configuration options, the topology file, setting options in example config files - this went bad a few times!).

**Pull request closed without merging**
